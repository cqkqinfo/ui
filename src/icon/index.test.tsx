import React from 'react';
import Icon from './index';
import { render } from '@testing-library/react';

describe('图标', () => {
  it('渲染', () => {
    const { baseElement } = render(<Icon name={'kq-add'} />);

    expect(baseElement).toMatchSnapshot();
  });
});
