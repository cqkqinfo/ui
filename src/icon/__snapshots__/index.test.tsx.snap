// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`图标 渲染 1`] = `
<body>
  <div>
    <div
      class="wrap"
      id="id0"
    />
  </div>
</body>
`;
